trieste/acquisition/rule.py

```diff
@@ -382,9 +404,9 @@ def __init__(
             builder = builder.using(OBJECTIVE)
 
         # even though we are only using batch acquisition functions
-        # there is no need to batchify the optimizer if our batch size is 1
+        # there is no need to batchify_joint the optimizer if our batch size is 1
```

Comment:
why not the same for batchify_vectorize?

Reply:
Ive not added functionality for async BO